src/pages/callouts/[id].vue

```diff
@@ -132,7 +132,7 @@ import InfoMessage from '../../components/InfoMessage.vue';
 
 import 'formiojs/dist/formio.form.css';
 import { useRoute } from 'vue-router';
-import CalloutStatus from '../../components/callout/CalloutStatus.vue';
+import ItemStatusComponent from '../../components/item/ItemStatus.vue';
```

Comment:
The convention is to give component imports the same name as their file, so this should be
```ts
import ItemStatus from '../../components/item/ItemStatus.vue';
```

Reply:
Problem was: There is already an `ItemStatus` in `common`. Renamed the component to something a bit more descriptive

Reply:
Ahh sorry I missed this :+1: